examples/plotting/trefoil.py

```diff
@@ -1,3 +1,13 @@
+''' This example plots Radiation Warning Symbol (Trefoil). It demonstrates
```

Comment:
```suggestion
''' This example plots a Radiation Warning Symbol (Trefoil). It demonstrates
```

Reply:
This example show the international symbol for radiation warning, so if anything, I would change this to "the". Also, I would change "plots" to "shows" or "visualizes" or something like that.